src/site/content/en/lighthouse-performance/total-blocking-time/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Total Blocking Time
+description: |
+  Learn about Lighthouse's Total Blocking Time metric and
+  how to measure and optimize it.
+web_lighthouse:
+  - total-blocking-time
+date: 2019-10-09
+---
+
+Total Blocking Time (TBT) is one of the metrics tracked in the **Performance** section
+of the Lighthouse report. Each metric captures some aspect of page load speed.
+
+The Lighthouse report displays TBT in milliseconds:
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="total-blocking-time.jpg" alt="A screenshot of the Lighthouse Total Blocking Time audit">
+</figure>
+
+## What TBT measures
+
+TBT measures the total amount of time that a page is blocked from responding to user input,
+such as mouse clicks, screen taps, or keyboard presses. The sum is calculated by adding
+the duration of all [long tasks] between [First Contentful Paint][fcp] and [Time to Interactive][tti].
```

Comment:
`[long tasks]` missing a link?

technically, it's summing all of the *blocking portion* of all long tasks between FCP and TTI, i.e. how much longer than 50ms was each long task. not sure how to word that eloquently though

Reply:
Actually this is fancy shorthand syntax! Similar to object shorthand `{name}` in JS. Surma used this in a post a few weeks back and I had the exact same response as you.

Reply:
Glad I asked for your review. I didn't catch that *blocking portion* stuff. When in doubt, I just add an example to explain complicated concepts. If the updated text looks good to you please resolve this convo.

Reply:
woah neat well TIL! and cool this looks great 👍